### PR TITLE
Pulled changes over from #845 by @GaryRidgway.

### PR DESCRIPTION
**Pull request by pyrello**
# To test
```
yarn storybook
```
- Verify no deprecation warnings.

### CODE REVIEW

**Comment by GaryRidgway** [comment]
Sheesh, this was stupid. I had to remove divides as well because they were depreciated in whatever version of sass that Sitenow was using. So i just wrote my own.

File: src/assets/scss/_utilities.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@use "sass:math";
```
Comment (bspeare) [inline src/assets/scss/_utilities.scss]: Do we still need this line?

**Comment by bspeare** [comment]
Let's make sure we update these once it is moved to uids_base: https://github.com/uiowa/uiowa/search?q=containerClamp.

File: src/assets/scss/_utilities.scss (hunk 1 above)
Comment (pyrello) [reply src/assets/scss/_utilities.scss]:
```suggestion
```
I don't think so.

File: src/assets/scss/_utilities.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (pyrello) [inline src/assets/scss/_utilities.scss]:
```suggestion
```

**Review by bspeare — APPROVED** [review]
These look like the expected values.   Code reviewed, not tested.  Let's not forget https://github.com/uiowa/uids/pull/853#issuecomment-1400578987 and also updating the 3.x branch?

**Pull request merged**